.github/workflows/main.yml

```diff
@@ -134,51 +152,30 @@ jobs:
                   echo "matrix=$matrix" >> $GITHUB_OUTPUT
 
     slither:
-        if: needs.changed-files.outputs.any_changed == 'true'
-        needs: [changed-files, set-matrix, deps]
+        if: contains(needs.changed-files.outputs.modified_files, 'lean-imt')
```

Comment:
somehow slither was buggy when the monorepo combines contracts that are compiled with either hardhat or forge. As slither in the ci is new, and because eventually we'll have only forge, I suggest restricting the ci check to lean-imt as a temp fix.